src/main/java/org/opentripplanner/api/mapping/PlannerErrorMapper.java

```diff
@@ -45,9 +45,6 @@ public static PlannerError mapMessage(RoutingError domain) {
       case WALKING_BETTER_THAN_TRANSIT:
         api = new PlannerError(Message.TOO_CLOSE);
         break;
-      case SYSTEM_ERROR:
```

Comment:
What happens instead?

Reply:
The SYSTEM_ERROR is not returned, it is handled by the GraphQL framework and reported in the errors section of the root response - if we does not know what happen, then we should not return a partial result either - it might be corrupt.